routes/actionsRouter.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
+const route = express.Router();
+
+const actionModel = require("../data/helpers/actionModel");
+//middleware
+const checkAction = require("../common/checkAction");
+
+function generalErr(err, res) {
+  res
+    .status(500)
```

Comment:
Luis recommended creating something like a response object `const responseStatus = { serverErr: 500 }` and then using it on 10 `.status(responseStatus.serverErr)` and doing it for all of your requests to avoid having `magic numbers`.